step.yml

```diff
@@ -0,0 +1,45 @@
+title: Flutter Installer
+summary: This step will install the selected version of Flutter SDK if flutter is not yet available on the system.
+description: This step will install the selected version of Flutter SDK if flutter is not yet available on the system.
+website: https://github.com/bitrise-steplib/bitrise-step-flutter-installer
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-flutter-installer
+support_url: https://github.com/bitrise-steplib/bitrise-step-flutter-installer/issues
+
+type_tags:
+  - utility
+
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+
+deps:
+  brew:
+  - name: git
+  apt_get:
+  - name: git
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/bitrise-step-flutter-installer
+
+inputs:
+  - version: v1.0.0
```

Comment:
ouh, do not lock a specific version, either use latest, master or something
i think by default we shlould install the latest stable version of the sdk if the user does not define anything else